data-prepper-plugins/s3-sink/src/main/java/org/opensearch/dataprepper/plugins/sink/configuration/ObjectOptions.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.sink.configuration;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.constraints.NotNull;
+
+/**
+ * An implementation class of file pattern configuration Options
+ */
+public class ObjectOptions {
+    private static final String DEFAULT_KEY_PATTERN = "logs-${yyyy-MM-dd hh:mm:ss}";

```

Comment:
Please use a default value without spaces. So add a `T` instead of a space. This is inline with the ISO-8601 standard.

Also, I think we might not want logs in the name. Maybe "events" instead.

```
events-${yyyy-MM-ddThh:mm:ss}
```

Reply:
Updaded name_pattern as per ISO-8601 standard as below.
name_pattern: event-%{yyyy-MM-dd'Thh-mm-ss'}